test/rollup.treeshake.config.js

```diff
@@ -98,7 +98,7 @@ export default [
 		]
 	},
 	{
-		input: 'test/treeshake/index.webgpu.nodes.js',
+		input: 'test/treeshake/index.webgpu.js',
```

Comment:
`three.webgpu.nodes.js` is unused, and I can no longer find references to `three/nodes`.